### PR TITLE
release: 0.3.1

### PR DESCRIPTION
**Pull request by NoNameProvided**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+- added Eslint for linting
+- updated CI configuration
+- removed some unused dev dependencies
+- updated dependencies to latest version
+
+#### Fixed
+
+- circular dependency fixed
+- dev dependencies removed from package.json before publishing (no more security warnings)
+- transformer oder is deterministic now (#231)
```
Comment (saulotoledo) [inline CHANGELOG.md:24]: typo: order

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+- fix prototype pollution issue (#367)
+
```
Comment (saulotoledo) [inline CHANGELOG.md:26]: minor doc issues: https://github.com/typestack/class-transformer/pull/382/commits/915159b39fc63342d41853074f7af4b5bdd1f399

Comment (NoNameProvided) [reply CHANGELOG.md:26]: Thanks! Fixed in 9bd7408.

File: CHANGELOG.md (hunk 1 above)
Comment (NoNameProvided) [reply CHANGELOG.md:24]: Thanks! Fixed in 9bd7408.

**Pull request merged**